scripts/install/setup.sh

```diff
@@ -323,3 +323,6 @@ deploy_ready spin-deck "UI server"
 echo
 bold "Installation complete."
 echo
+bold "Make sure to Sign up for Spinnaker for GCP updates and announcements:"
```

Comment:
[minor] Perhaps just "Sign up for ..."? But fine either way.

Thanks!

Reply:
Thought I had fixed that in response to David suggesting the same thing.

Got it right this time: https://github.com/GoogleCloudPlatform/spinnaker-for-gcp/pull/105